PurchasesTests/LocalReceiptParsing/Builders/InAppPurchaseBuilderTests.swift

```diff
@@ -4,12 +4,13 @@
 //
 
 import Foundation
-import XCTest
 import Nimble
+import XCTest
 
 @testable import RevenueCat
 
 class InAppPurchaseBuilderTests: XCTestCase {
+    // swiftlint:disable force_try
```

Comment:
This one I think it does make sense to locally disable 👍 because in many cases it'll remind people to use `XCTUnwrap `

Reply:
yep, totally agree with you